projects/optic/src/commands/capture/storage.ts

```diff
@@ -34,15 +34,23 @@ export async function getCaptureStorage(filePath: string): Promise<string> {
 }
 
 export class GroupedCaptures {
```

Comment:
So I'm storing both interactions and hars here because we only have fns to convert postman -> interactions.

In the future we'd probably want everything to be directly stored as interactions but there's some other work to do to support this